src/Microsoft.ML.Data/Transforms/ConcatTransform.cs

```diff
@@ -90,19 +90,6 @@ public bool TryUnparse(StringBuilder sb)
 
         public sealed class Arguments : TransformInputBase
         {
-            public Arguments()
```

Comment:
>public Arguments() [](start = 12, length = 18)

I think something terribly wrong happen with your merge

Reply:
Yes, I am aware and fixing it via interactive rebase.

---
In reply to: [199600627](https://github.com/dotnet/machinelearning/pull/454#discussion_r199600627) [](ancestors = 199600627)